src/views/Browser.vue

```diff
@@ -325,10 +325,11 @@ export default {
         } else {
           return Object.values(state.api.error)
             .filter(e => e !== null)
+            .filter(e => e !== state.api.error.datasetsAnnotationsByCategory || !Object.values(e).every(value => value === null))
```

Comment:
I suggest you use flatMap here and Object.entries.

Something like:

```javascript
Object.entries(state.api.error).flatMap(([key, error]) => key === 'datasetsAnnptationsByCategory' ? Object.values(error) : [error])
```

Reply:
changed in commit c4eafa767ccb33eeb9bd48b16973fc6e16901bdf